webapp/views.py

```diff
@@ -506,6 +503,13 @@ def make_renewal(advantage, contract_info):
 
     renewal["renewable"] = False
 
+    if renewal["status"] == "done":
+        renewalModifiedDate = dateutil.parser.parse(renewal["lastModified"])
```

Comment:
I'd recommend catching errors here (wrapping this in a `try`) and handling it somehow (i.e. not marking it as recently renewd).
We try to keep our APIs sane in ua-contracts, but you never know when things can go wrong :)